CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ Each e2e profile needs to be configured for a different AWS account and a differ
 Below are the different commands which can be run in the root of the project directory.
 
 * Run `make` (This creates a standalone executable in the `bin/local` directory).
-* Run `make test` to run the unit tests.
+* Run `make run-unit-test` to run the unit tests.
```

Comment:
nit: maybe we should keep the old one but different between the two commands?
```
* Run `make run-unit-test` to run only Go unit tests.
* Run `make test` to run Go and Nodejs unit tests.
```

Reply:
I added old commands (`make test`).